### ISSUE TITLE
Coffescript/node/cake conflict

### ISSUE THREAD

**Issue by johnpaulashenfelter** [swannodette/textmate-clojure#6]
When I run the first demo (+ 4 5) ctrl-x I get

/usr/local/Cellar/coffee-script/0.9.2/lib/coffee-script/lib/cake.js:40 throw new Error("Cakefile not found in " + (process.cwd())); ^ Error: Cakefile not found in /Users/johnpaul/projects at /usr/local/Cellar/coffee-script/0.9.2/lib/coffee-script/lib/cake.js:40:15 at path:81:19 at node.js:757:9

Any ideas how to get to the right cake? I'm also using rvm to make things harder :)

**Comment by swannodette** [comment]
I'm looking into the RVM issue today, but fixing name conflicts between the cake's seems out of scope for us. Will mention it to the cake maintainers.

**Comment by johnpaulashenfelter** [comment]
I need to just RTFM! I was getting the wrong gem environment because of RVM -- I used the instructions at http://rvm.beginrescueend.com/integration/textmate/ for setting up textmate rvm integration and now everything is working fine (though I also had to remove my old hardcoded path in textmate which wasn't picking up the rvm path". I also had cake.js in my path from a homebrew-based node.js/coffee-script install.

So basically, I had to 
- RTFM for rvm textmate integration
- install cake gem in the ruby being used in textmate
- remove hardcoded path info in textmate so rvm could do it's job
